usr/lib/systemd/system/apisix.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=apisix
+Conflicts=apisix.service
+After=network.target
+
+[Service]
+Type=forking
+WorkingDirectory=/usr/bin
+ExecStart=/usr/bin/apisix start
+ExecStop=/usr/bin/apisix stop
+ExecReload=/usr/bin/apisix restart
```

Comment:
Should be `/usr/bin/apisix reload`.
You can refer the doc: https://www.freedesktop.org/software/systemd/man/systemd.service.html

Reply:
Thanks for the information!